Resources/Prototypes/Recipes/Reactions/chemicals.yml

```
@@ -515,3 +515,15 @@
       amount: 1
   products:
     Tazinide: 1
+
+- type: reaction
+  id: ArtifactGlue
+  reactants:
+    SpaceGlue:
+      amount: 2
+    Artifexium:
+      amount: 2
+    Uranium:
+      amount: 1
+  products:
+    ArtifactGlue: 1 # 5 to 1. Pretty clear for artifact glue i suppose?..
```

```suggestion
    ArtifactGlue: 5
```
You shouldn't decrease volume after a reaction, or else it causes massive temperature fluctuations. It's also just a bit wasteful of the materials.